docs/pages/platform/webhooks.mdx

```diff
@@ -874,18 +885,93 @@ type NotificationEvent = {
 };
 
 // Example
-const notificationEvent = {
+const threadNotificationEvent = {
   type: "notification",
   data: {
     channel: "email",
     kind: "thread",
     projectId: "my-project-id",
     roomId: "my-room-id",
-    userId: "my-user-id
+    userId: "my-user-id",
     threadId: "my-thread-id",
     inboxNotificationId: "my-inbox-notification-id",
     createdAt: "2021-10-06T01:45:56.558Z",
-    },
+  },
+};
+```
+
+##### TextMention notification
+
+When using [Text editor](/docs/product/text-editor), an event is triggered 30
+minutes after a user has been mentioned in a text and has not seen the text
+mention. This is the Liveblocks `textMention` notification kind.
+
+```ts
+// Schema
+type TextMentionNotificationEvent = {
+  type: "notification";
+  data: {
+    channel: "email";
+    kind: "textMention";
+    projectId: string;
+    roomId: string;
+    userId: string;
+    mentionId: string;
+    inboxNotificationId: string;
+    createdAt: string;
+  };
+};
+
+// Example
+const textMentionNotificationEvent = {
+  type: "notification",
+  data: {
+    channel: "email",
+    kind: "textMention",
+    projectId: "my-project-id",
+    roomId: "my-room-id",
+    userId: "my-user-id",
+    mentionId: "my-mention-id",
+    inboxNotificationId: "my-inbox-notification-id",
+    createdAt: "2021-10-06T01:45:56.558Z",
+  },
+};
+```
+
+##### Custom notification
+
+An event is triggered 30 minutes after the user has been notified of a custom
+event and has not seen the notification. All custom notification `kinds` are
+prefixed with `$` and are manually by you on the server. Learn more about
+[triggering custom notifications](/docs/api-reference/rest-api-endpoints#post-inbox-notifications-trigger).
```

Comment:
I've changed the link from this:
```md
Learn more about triggering custom notifications [here](/docs/...).
```
To this:
```md
Learn more about [triggering custom notifications](/docs/...).
```

**Interesting fact**
If you describe where a link goes inside the `<a>` tag, instead of using a word like "here", it makes links more accessible and understandable (e.g. for those with screen readers). It's also better for SEO, as then Google can understand the meaning of our links better!

Reply:
TIL, thank you!